gluten-core/src/main/java/io/glutenproject/vectorized/JniWorkspace.java

```diff
@@ -29,6 +30,13 @@ private JniWorkspace(String rootDir) {
       this.jniLibLoader = new JniLibLoader(workDir);
       this.jniResourceHelper = new JniResourceHelper(workDir);
       LOG.info("JNI workspace {} created in root directory {}", workDir, rootDir);
+      Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+        try {
+          FileUtils.deleteDirectory(created.toFile());
+        } catch (Exception e) {
+          throw new RuntimeException(e);
+        }
+      }));
```

Comment:
nit: can we use FileUtils.deleteDirectoryOnExit(...) ?

Reply:
`FileUtils.deleteDirectoryOnExit` is a private method.
I also try `FileUtils.forceDeleteOnExit`, but not work.

Reply:
Aha. Thanks for clarifying. But `FileUtils.forceDeleteOnExit`, this one should be suitable as long as it does what its doc says. But it's OK to keep the code as is.